src/lib/aws/common/stack.ts

```diff
@@ -104,7 +106,8 @@ export class CommonStack extends Stack {
   protected determineStageContexts() {
     const stage = this.node.tryGetContext('stage')
     const stageContextPath = this.node.tryGetContext('stageContextPath') || 'cdkEnv'
-    const stageContextFilePath = `${appRoot.path}/${stageContextPath}/${stage}.json`
+    const stageContextFilePath = path.join(appRoot.path, stageContextPath, `${stage}.json`)
```

Comment:
If we are changing the format, we need to do it across the whole repository. We can't have different formats in the repository which is being introduced here.

Reply:
I will change in other places as well to ensure we don't handcraft path manually and update to use `path.join`, as this is `nodejs` sdk to join paths segments together using the platform-specific separator as a delimiter and normalise them as well. 

see [documentation](https://nodejs.org/api/path.html#pathjoinpaths)